docs/_providers/doh.md

```diff
@@ -0,0 +1,37 @@
+---
+name: DNS-over-HTTPS
+title: DNS-over-HTTPS Provider
+layout: default
+jsId: DNSOVERHTTPS
+---
+# DNS-over-HTTPS Provider
+
+This is a read-only/fake "registrar" provider. Sometimes you're stuck with a registrar without an API, instead of using the `NONE` registrar, use the `DNSOVERHTTPS` registrar to do a DNS NS lookup and check the returned NS servers match the expected servers. This "registatrar" can't update them for you, but will at least let you know they are not what you're expecting.
+
+## Configuration
+In your `creds.json` file, you must provide the host of your DNS-over-HTTPS provider.
+
+{% highlight json %}
+{
+  "DNS-over-HTTPS": {
+    "host": "cloudflare-dns.com"
+  }
+}
+{% endhighlight %}
+
+Some common options are `cloudflare-dns.com` [Cloudflare](https://developers.cloudflare.com/1.1.1.1/dns-over-https), `9.9.9.9` [Quad9](https://www.quad9.net/about/), and `dns.google` [Google Public DNS](https://developers.google.com/speed/public-dns/docs/doh)
```

Comment:
How about a default?  It would be nice to not require a creds.json file if not needed (BIND and others).